### PR TITLE
Fix some issues in the system tests

### PR DESCRIPTION
**Pull request by ctron**
* If the route is not found, throw a more helpful exception.
* Combine the two setup methods, as they do rely on an order
  but @BeforEach does not honor any order
* Fix a race condition when deleting an IoTProject. It relies
  on an initial set of data, which might already be gone, as the
  deletion is on progress.

### CODE REVIEW

**Comment by ctron** [comment]
@enmasse-ci run tests profile=iot testcase=io.enmasse.systemtest.iot.isolated.IoTProjectManagedTest.testCreate

**Comment by kornys** [comment]
@enmasse-ci run tests profile=iot testcase=io.enmasse.systemtest.iot.isolated.IoTProjectManagedTest#testCreate

File: systemtests/src/main/java/io/enmasse/systemtest/utils/IoTUtils.java [hunk 1]
```diff
@@ -177,7 +177,15 @@ public static void waitForIoTProjectReady(Kubernetes kubernetes, IoTProject proj
         assertThat(project.getStatus().getDownstreamEndpoint().getPort(), not(is(0)));
     }
 
-    private static void waitForIoTProjectDeleted(Kubernetes kubernetes, IoTProject project) throws Exception {
+    public static boolean isIoTInstalled(Kubernetes kubernetes) {
+        return kubernetes.getCRD(IoTCrd.project().getMetadata().getName()) != null;
```
Comment (lulf) [inline systemtests/src/main/java/io/enmasse/systemtest/utils/IoTUtils.java]: This will always be true, the CRD is always created when installing.

Comment (ctron) [reply systemtests/src/main/java/io/enmasse/systemtest/utils/IoTUtils.java]: Indeed. That came back by accident. It is removed again.

**Comment by ctron** [comment]
@enmasse-ci run tests

**Comment by ctron** [comment]
@enmasse-ci run tests

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared testcase=io.enmasse.systemtest.shared.standard.web.FirefoxConsoleTest#testAddressStatus

**Pull request merged**